source/LitLiterary/LitLiteracy.Tests/Services/RepositoryTests.cs

```diff
@@ -0,0 +1,13 @@
+using NUnit.Framework;
+
+namespace LitLiteracy.Tests.Services
+{
+    public class RepositoryTests
```

Comment:
Perhaps it should be named `PlaceholderTests`. Since we plan on using Moq to fake the repository contexts during testing. I don't suspect much value would come from testing an actual or mocked repository.

Reply:
That's fair, I'll rename it. Thanks